### ISSUE TITLE
Improve react blink component

### ISSUE THREAD

**Issue by elangosundar** [10secondsofcode/react-blink-text#6]
Improve some feature or suggestions for react blink components.

**Comment by morijenab** [comment]
Is this the file that need to be work on?
https://github.com/10secondsofcode/react-blink-text/blob/master/lib/ReactBlinkText.js

**Comment by elangosundar** [comment]
@Morteza-Jenabzadeh Nope, just improve this component https://github.com/10secondsofcode/react-blink-text/blob/master/src/ReactBlinkText.js

**Comment by allysonjeronimo** [comment]
Is there className param support on Blink? Can I implement it. Might be useful in case we want to use a custom css class instead of the color, font or other built in params.

**Comment by 10-seconds** [comment]
> Is there className param support on Blink? Can I implement it. Might be useful in case we want to use a custom css class instead of the color, font or other built in params.

sure
